dbt_platform_helper/templates/environment-pipelines/main.tf

```diff
@@ -0,0 +1,53 @@
+# {% extra_header %}
+# {% version_info %}
+locals {
+  platform_config    = yamldecode(file("../../platform-config.yml"))
+  all_pipelines      = local.platform_config["environment_pipelines"]
+  pipelines = { for pipeline, config in local.platform_config["environment_pipelines"] : pipeline => config if config.account == "{{ aws_account }}" }
+  environment_config = local.platform_config["environments"]
+}
+
+provider "aws" {
+  region                   = "eu-west-2"
+  profile                  = "{{ aws_account }}"
+  alias                    = "{{ aws_account }}"
+  shared_credentials_files = ["~/.aws/config"]
+}
+
+terraform {
+  required_version = "~> 1.8"
+  backend "s3" {
+    bucket         = "terraform-platform-state-{{ aws_account }}"
+    key            = "tfstate/application/{{ application }}-pipelines.tfstate"
+    region         = "eu-west-2"
+    encrypt        = true
+    kms_key_id     = "alias/terraform-platform-state-s3-key-{{ aws_account }}"
+    dynamodb_table = "terraform-platform-lockdb-{{ aws_account }}"
+  }
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5"
+    }
+  }
+}
+
+
+module "environment-pipelines" {
+  source = "git::https://github.com/uktrade/terraform-platform-modules.git//environment-pipelines?depth=1&ref={{ terraform_platform_modules_version }}"
+  #  source = "../../../terraform-platform-modules/environment-pipelines"
```

Comment:
I'm not sure we should keep this in the autogenerated file...
```suggestion
```

Reply:
is it maybe useful to leave for people to use? Or maybe in the case of 1073, we would add a command for this command to generate the main.tf with a realitive source instead? Just thinking out loud here

Reply:
I don't really mind, it's not a deal breaker, but it feels untidy to leave it in our production code.